cluster.tf

```diff
@@ -133,21 +135,25 @@ resource "google_container_cluster" "primary" {
       }
     }
 
-    dynamic "daily_maintenance_window" {
```

Comment:
Shouldn't this still be a dynamic block?

Reply:
Yes, let me updated that